my-app/src/components/Body/Content/BlogView/BlogView.jsx

```diff
@@ -0,0 +1,90 @@
+    createData('Design: A Survival Guide for Beginners', 'Posted 3 days ago', 'Published', 120, "views", ""),
+];
+
+const BlogView = () => {
+    const classes = useStyles();
+
+    return (
+        <div className="blogView">
+            <TableContainer className={classes.tableContainer} component={Paper}>
+                <Table className={classes.table} aria-label="simple table">
+                    <TableHead>
+                        <TableRow>
+                            <TableCell className={classes.tableCell} align="left">Title</TableCell>
+                            <TableCell className={classes.tableCell} align="left">Status</TableCell>
+                            <TableCell className={classes.tableCell} align="left">Stats</TableCell>
+                            <TableCell className={classes.tableCell} align="right">
+                                <Button startIcon={<PenToolWhite/>} className={classes.button}>Add new</Button>
+                            </TableCell>
+                        </TableRow>
+                    </TableHead>
+                    <TableBody>
+                        {rows.map((row) => (
+                            <TableRow key={row.name} className={classes.row}>
+                                <TableCell component="th" scope="row" className={classes.cell}>
+                                    <Typography variant="h3" className={classes.name}>
+                                        {row.name}
+                                    </Typography>
+                                    <Typography variant="h5" className={classes.subtext}>
+                                        {row.posted}
+                                    </Typography>
+                                </TableCell>
+                                <TableCell align="center" className={classes.cell}>
+                                <span className={classes.status + " " + row.status}>
+                                    {row.status}
+                                </span>
+                                </TableCell>
+                                <TableCell align="center" className={classes.cell}>
+                                    <div className={classes.container}>
+                                        <span className={classes.stats}>{row.stats}</span>
+                                        <span className={classes.views}>{row.views}</span>
+                                        {row.stats > 0
+                                            ? <span> <img src={arrow} alt="arrow"/></span>
+                                            : <span> </span>
+                                        }
+                                    </div>
+                                </TableCell>
+                                <TableCell align="right" className={classes.cell}>
+                                    <div className={classes.dots}>
```

Comment:
use already exist component for this dots or create own

Reply:
done (create own)